examples/tvdemo/gadgets.cpp

```diff
@@ -26,7 +26,6 @@
 #include <ctype.h>
 #include <strstrea.h>
 #include <iomanip.h>
-#include <malloc.h>

```

Comment:
I guess malloc.h does not exist in FreeBSD?

Reply:
`malloc.h` causes no issues for me in FreeBSD 12.1 with GCC 9.3. What error do you see, @rmorales87atx?

Reply:
I was getting this message, compiling with clang:
```
/usr/include/malloc.h:3:2: error: "<malloc.h> has been replaced by <stdlib.h>"
#error "<malloc.h> has been replaced by <stdlib.h>"
```

I guess it "works" for you since you specifically installed GCC 🤔

Reply:
Okay, no problem. Unfortunately, this header is needed when compiling with Borland C++, so I'll have to find another solution.

Reply:
I suspected that would be an issue, tbh... The overall better solution is to add detection to avoid including `malloc.h` under clang+FreeBSD

Reply:
Still working for me... my malloc.h is as follows:

![image](https://user-images.githubusercontent.com/20713561/92625413-e6560700-f2c8-11ea-8f51-6fe55db6567a.png)

But one of the reasons why `malloc.h` is needed is for `alloca`. Do you know where it is defined in FreeBSD?

Reply:
According to `man alloca` it should be in `stdlib.h`

![image](https://user-images.githubusercontent.com/3020857/92625997-f81ab880-f28e-11ea-94c5-b37c171a869b.png)

There's a bug report that's still "open" which suggests FreeBSD's `malloc.h` should be "changed back": https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=155429

I wonder if installing gcc is what changed your `malloc.h` - maybe not the gcc package itself but some related package

Reply:
The date in my header is more recent than the date of the commit referenced by that bug, https://svnweb.freebsd.org/changeset/base/360984. So maybe it has been actually fixed.

I will fix it in Turbo Vision anyway.

Reply:
Thanks for the feedback and review, I do appreciate your time. My installation is still -RELEASE and not -STABLE so that's very likely why I still have an "old" `malloc.h`.

In any case I'm glad to have things working "for the moment" until more proper fixes come through 😃